src/main/java/uk/org/okapibarcode/output/Java2DRenderer.java

```diff
@@ -67,10 +67,28 @@ public class Java2DRenderer implements SymbolRenderer {
      * @param ink the ink (foreground) color
      */
     public Java2DRenderer(Graphics2D g2d, double magnification, Color paper, Color ink) {
+        this(g2d, magnification, paper, ink, 0);
+    }
+
+    /**
+     * Creates a new Java 2D renderer with the specified orientation. If the specified paper color is {@code null}, the symbol is drawn without clearing the
+     * existing {@code g2d} background.
+     *
+     * @param g2d the graphics to render to
+     * @param magnification the magnification factor to apply
+     * @param paper the paper (background) color (may be {@code null})
+     * @param ink the ink (foreground) color
+     * @param orientation the orientation of the symbol (0, 90, 180, or 270)
+     */
+    public Java2DRenderer(Graphics2D g2d, double magnification, Color paper, Color ink, int orientation) {
```

Comment:
Let's call this `rotation` instead of `orientation` across the board, and document in all JavaDoc that the rotation direction is clockwise.

Reply:
Resolved